docker/prod/Dockerfile

```diff
@@ -0,0 +1,70 @@
+FROM python:3.7-alpine3.10 AS base
```

Comment:
Can we upgrade to `python:3.7-alpine3.11`, or even better, `python:3.8-alpine3.11`?

Reply:
I think some of the stuff we are using don't oficially support python3.8 (and poetry checks that), at least that was the case a month ago when I started this. I will reverify and update whatever I can to the newest version.

Reply:
All seems working with py38 alpine 3.11